locales/en-US/strings.properties

```diff
@@ -5,25 +5,35 @@ bookmarked=Bookmarked
 synced=Synced from another device
 open=Open
 topic=Topic
+# LOCALIZATION NOTE: 'bookmark' is an action meant as 'add to bookmarks'
```

Comment:
Typically localization notes look like `LOCALIZATION NOTE (identifier):` to be clear what it's referring to. Also, we could probably avoid some of these types of notes with more explicit identifiers, e.g., `menu_action.bookmark` vs `type_label.synced`. Multi-word identifiers also make it much easier to search for the string in the code, e.g., "open" probably will have a bunch of false positive matches from a simple `grep`.